classes/reportbase.php

```diff
@@ -53,6 +53,11 @@ abstract class reportbase {
      */
     public $groupid = 0;
 
+    /**
+     * @array int $groupid
```

Comment:
Heh, this needs to be `@var array documentation-text-goes-here`

Reply:
You are right. This is because, to save time, I run pulls hurriedly before starting other tasks so that I can find the executed GHAs when I return to surveypro.